py/nightwatch/run.py

```diff
@@ -427,7 +427,7 @@ def make_plots(infile, basedir, preprocdir=None, logdir=None, rawdir=None, camer
             htmlfile = '{}/qa-guide-{:08d}.html'.format(expdir, expid)
             web_guide.write_guide_html(htmlfile, header, guidedata)
             print('Wrote {}'.format(htmlfile))
-        except FileNotFoundError:
+        except FileNotFoundError or OSError:
```

Comment:
I think the standard python syntax for this is
```
except (FileNotFoundError, OSError):
```

`FileNotFoundError or OSError` seem to evaluate to `FileNotFoundError` so it is unclear to me that this is actually also catching `OSError` too.  Or it might be a special case in the `except` syntax that I'm not aware of.  Please check and confirm.